frontend/src/components/summary/group/RFISummary.vue

```diff
@@ -981,139 +107,377 @@
+                :rules="rules.required"
+              />
+            </v-col>
+
+            <v-col class="py-0 cols-6">
+              <span class="summary-label pt-3"
+                >Is the wage increase part of a collective bargaining agreement for Direct Care Staff at the
+                facility?</span
+              >
+              <v-text-field
+                placeholder="Required"
+                class="summary-value"
+                :model-value="getValueString(rfiApp?.isBargainingAgreement)"
+                density="compact"
+                flat
+                variant="solo"
+                hide-details
+                required
+                :rules="rules.required"
+              />
+            </v-col>
+
+            <v-col class="py-0 cols-6">
+              <span class="summary-label pt-3"
+                >Has the facility lost or been unable to hire Direct Care Staff due to current wages?</span
+              >
+              <v-text-field
+                placeholder="Required"
+                class="summary-value"
+                :model-value="getValueString(rfiApp?.lossOfCareStaff)"
+                density="compact"
+                flat
+                variant="solo"
+                hide-details
+                required
+                :rules="rules.required"
+              />
+            </v-col>
+
+            <v-col class="py-0 cols-6">
+              <span class="summary-label pt-3"
+                >Is this creating challenges in maintaining the staff-to-child ratios required under the facility
+                licence?</span
+              >
+              <v-text-field
+                placeholder="Required"
+                class="summary-value"
+                :model-value="getValueString(rfiApp?.healthAndSafetyConcerns)"
+                density="compact"
+                flat
+                variant="solo"
+                hide-details
+                required
+                :rules="rules.required"
+              />
+            </v-col>
+          </v-row>
+        </div>
+
+        <v-col v-if="rfiApp?.feeIncreaseDueToWage == 1" class="col-12">
+          <v-row no-gutters class="d-flex">
+            <v-col
+              v-for="header in [
+                'Number of staff receiving wage increase',
+                'Direct Care staff role',
+                'Wage before increase',
+                'Wage after increase',
+                'Average hours per week at this facility',
+                'Date',
+              ]"
+              :key="header"
+              class="d-flex justify-start col-2 pa-0 pr-2"
+            >
+              <span class="summary-label">{{ header }}</span>
+            </v-col>
+          </v-row>
+
+          <v-row v-if="rfiApp?.wageList.length === 0" no-gutters class="d-flex">
+            <v-col v-for="n in 6" :key="n" class="d-flex justify-start col-2 pa-0 pr-2">
+              <v-text-field
+                placeholder="Required"
```

Comment:
Currently, the Required error text is blur. Can we update style in this file for placeholder? we should use deep instead of >>>. also, we need important for opacity.  (please use CCOFSummary page as reference).

Reply:
done